test/services/DateServiceSpec.scala

```diff
@@ -50,25 +51,15 @@ class DateServiceSpec extends TestSupport with FeatureSwitching {
     }
   }
 
-  "The getCurrentTaxYearEnd method when TimeMachineAddYear FS is on" should {
-    "return the next year if the current date is before april 6 and the timeMachineAddYears 1" in {
-      enable(TimeMachineAddYear)
-      TestDateService.getCurrentTaxYearEnd(LocalDate.parse("2018-03-29")) shouldBe 2019
+  "The getCurrentTaxYearEnd" should {
+    "return the current tax year" in{
+      disable(TimeMachineAddYear)
+      TestDateService.getCurrentTaxYearEnd shouldBe 2023
     }
-    "return the following year if current date is on or after april 6 and the timeMachineAddYears 1" in {
+    "return next tax year when time machine is enabled" in {
       enable(TimeMachineAddYear)
-      TestDateService.getCurrentTaxYearEnd(LocalDate.parse("2018-04-06")) shouldBe 2020
+      TestDateService.getCurrentTaxYearEnd shouldBe 2024
```

Comment:
this would not work next year, as mentioned in the prev comment, you would need to calculate year